acestep/ui/gradio/events/training/lora_training.py

```
@@ -0,0 +1,304 @@
+    lora_dropout: float,
+    learning_rate: float,
+    train_epochs: int,
+    train_batch_size: int,
+    gradient_accumulation: int,
+    save_every_n_epochs: int,
+    training_shift: float,
+    training_seed: int,
+    lora_output_dir: str,
+    resume_checkpoint_dir: str,
+    training_state: Dict,
+    progress=None,
```

Ruff (0.15.0)

[warning] Unused function argument: `progress`

(ARG001)